src/tests/ftest/util/ServerUtils.py

```diff
@@ -33,18 +33,126 @@
 import signal
 import fcntl
 import errno
+import yaml
 
 from avocado.utils import genio
 
 sessions = {}
 
+DEFAULT_YAML_FILE = "utils/config/examples/daos_server_sockets.yml"
+AVOCADO_YAML_FILE = "utils/config/daos_avocado_test.yml"
```

Comment:
I don't see this file included here, did I miss it?  If we need such a file it should go in the tests/ftest directory.

Reply:
This file is created during tests so it will not be available by default.